src/main/interface/settings.c

```diff
@@ -1122,6 +1122,7 @@ const clivalue_t valueTable[] = {
 #endif
 
 // PG_SYSTEM_CONFIG
+    { "system_configured",          VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_SYSTEM_CONFIG, offsetof(systemConfig_t, configured) },
```

Comment:
Not sure if we even want to expose this here. It's not settable, but it will show up in every `diff`.

Reply:
My bad. I've added this for debugging.